server.go

```diff
@@ -63,6 +63,10 @@ type ServerParams struct {
 	// to be used when querying the identity manager API.
 	IdentityAPIUsername string
 	IdentityAPIPassword string
+
+	// IdentityAPIKey holds the key to use for agent login to the identity
+	// manager API.
+	IdentityAPIKey *bakery.KeyPair
```

Comment:
Let's call this KeyPair or PrivateKey, as it's not specifically for the identity manager.